CHANGES.md

```diff
@@ -24,6 +24,12 @@ OpenSSL 3.1
 
 ### Changes between 3.0 and 3.1 [xx XXX xxxx]
 
+ * Add mac salt length configure for the pkcs12 command.
+   Also change the default value of mac salt length.
+   This improves the security.
```

Comment:
I don't think this line and the one before are worthwhile.  Just the first.

Reply:
Updated, thanks.